projects/notebooks/visualizations/rhods-notebooks-performance/models/lts.py

```diff
@@ -37,15 +37,16 @@ class BenchmarkMeasures(matbench_models.ExclusiveModel):
 
 class Results(matbench_models.ExclusiveModel):
     benchmark_measures: BenchmarkMeasures
+    regression: NotebookPerformanceRegression
 
 NotebookPerformanceKPIs = matbench_models.getKPIsModel("NotebookPerformanceKPIs", __name__, kpi.KPIs, kpi.NotebookPerformanceKPI)
+NotebookPerformanceRegression = matbench_models.RegressionResult
 
 class Payload(matbench_models.ExclusiveModel):
     schema_name: matbench_models.create_schema_field("rhods-notebooks-perf") = Field(alias="$schema")
     metadata: Metadata
     results: Results
     kpis: Optional[NotebookPerformanceKPIs]
-    regression_results: Optional[Any]
```

Comment:
can you let this here, in the `Payload`? in MatrixBenchmarking, overall, the `results` are workload specific, but the `payload/metadata/kpis` have a predefined structure (that can be extended, but the core is always the same) , so the regression results belong there. (I need to [browse these entries](https://github.com/openshift-psap/matrix-benchmarking/blob/main/matrix_benchmarking/upload_lts.py#L163) to upload them as KPIs)

Reply:
Gotcha, I'll move them under the `Payload`